app/markets/HistorySection.tsx

```diff
@@ -27,13 +24,8 @@ export const HistorySection = () => {
         </TabHeader>
         <TabBody className="my-4">
           <TabPanel>
-            <div className="bg-surface-surface-2 h-44 rounded-8">
-              <div className="flex items-center justify-center h-full">
-                <div className="flex flex-col items-center space-y-2">
-                  <Icon name="filter" size={24} />
-                  <p>Activity coming soon</p>
-                </div>
-              </div>
+            <div className="rounded-8">
```

Comment:
WDYT about adding a `className` to the `ActivityTable` component and remove this `div` element here?

Reply:
we can remove this div 👍